test/graphql/Offer/queries/getApprovedOffers.test.ts

```diff
@@ -38,6 +39,7 @@ describe("getApprovedOffers", () => {
   let approvedByExtensionBothOffer: Offer;
   let approvedByGraduadosBothOffer: Offer;
   let approvedByGraduadosAndExtensionBothOffer: Offer;
+  let expiredApprovedByGraduadosAndExtensionBothOffer: Offer;
```

Comment:
creo que seria "approved by both him and her" no "approved by him and her both"

Reply:
sobre los de arriba, que singifica "approved by graduados both offer"? quienes both?

Reply:
No lo hice yo pero asumo que la offer es para ambos y fue aprobado por ambos

Reply:
si, habria que ponerles nombres que no esten abiertos a ese "asumo"

Reply:
las 2 primeras variables las llamaria

"offer for students approved by extension"
"offer for graduates approved by graduados"

para seguir la misma estructura de nombres